From fd09dfba24a516243195878f6b97e5ae335861c8 Mon Sep 17 00:00:00 2001
From: Orlando Vazquez <ovazquez@gmail.com>
Date: Wed, 7 Sep 2016 12:39:27 -0700
Subject: [PATCH] CNAPI-668 CNAPI test uses `moray.client._emitResults`,
 causing test to fail, however it's undeeded since the mocked `findObjects`
 emits records itself

---
 test/model/test-model-platform.js | 4 ----
 test/model/test-model-server.js   | 4 ----
 2 files changed, 8 deletions(-)

diff --git a/test/model/test-model-platform.js b/test/model/test-model-platform.js
index 484c235..7137311 100644
--- a/test/model/test-model-platform.js
+++ b/test/model/test-model-platform.js
@@ -83,10 +83,6 @@ function testListPlatformsAll(test) {
                   '20150819T020110Z': { latest: true }});
             test.done();
         });
-
-        setTimeout(function () {
-            moray.client._emitResults(expSearchResults);
-        }, 100);
     });
 }
 
diff --git a/test/model/test-model-server.js b/test/model/test-model-server.js
index eae0bf1..7d47a6d 100644
--- a/test/model/test-model-server.js
+++ b/test/model/test-model-server.js
@@ -166,8 +166,6 @@ function testListServersByUuids(test) {
 
             test.done();
         });
-
-        moray.client._emitResults(expSearchResults);
     });
 }
 
@@ -228,8 +226,6 @@ function testListServersSetup(test) {
                 'moray history should match');
             test.done();
         });
-
-        moray.client._emitResults(expSearchResults);
     });
 }
 
-- 
2.21.0

